kobo/apps/subsequences/integrations/google/google_transcribe.py

```diff
@@ -74,6 +74,11 @@ def begin_google_operation(
         submission_uuid = self.submission.submission_uuid
         flac_content, duration = content
         total_seconds = int(duration.total_seconds())
+        # Check if the user's organization has available usage credits
+        org = Organization.get_from_user_id(self.user.pk)
+        if get_organization_remaining_usage(org, 'seconds') < total_seconds:
+            raise UsageLimitExceeded
```

Comment:
I think we said we're going to leave blocking excess usage for later, right? Same question goes for google translate below.

Reply:
Yes, right, I can revert these and bring them back later